### PR TITLE
Adding option to check if any modal is being displayed.

### PR DESCRIPTION
**Pull request by dryganets**

### CODE REVIEW

**Comment by Fitzpasd** [comment]
This should include some updates to the docs.

File: src/common/Interfaces.ts [hunk 1]
```diff
@@ -103,6 +103,7 @@ export abstract class Modal {
     abstract show(modal: React.ReactElement<Types.ViewProps>, modalId: string, options?: Types.ModalOptions): void;
     abstract dismiss(modalId: string): void;
     abstract dismissAll(): void;
+    abstract anyDisplayed(): boolean;
```
Comment (erictraut) [inline src/common/Interfaces.ts]: Keeping with the naming conventions in ReactXP, this should be called "isAnyDisplayed". I'm not sure this functionality requires a new method. We could allow users to specify an empty string to "isDisplayed" if they want to query whether any modal is displayed.

Comment (erictraut) [reply src/common/Interfaces.ts]: Also, any new exposed functionality should update the docs and the RXPTest sample.

**Comment by dryganets** [comment]
Updated the doc and API to what Eric proposed.
Now if modalId is undefined we check if we return true if any dialog is displayed.

**Comment by erictraut** [comment]
This change needs to be documented in Modal.md and added to the RXPTest app. I can do it later this week when I'm back from vacation or you can do it in the meantime.

**Comment by erictraut** [comment]
Never mind, I see that you added this to the documentation already. I'll add the RXPTest later.

**Pull request merged**